.github/CONTRIBUTING.md

```diff
@@ -755,7 +754,289 @@ It can always be reopened if it was a mistake.
+Here you should use the [ `TestHelper` ](https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/testing/test-helper.ts) property `ionicModuleForTest` like so: 
+
+``` 
+
+const createComponent = createComponentFactory({
+  component: RadioComponent,
+  imports: [TestHelper.ionicModuleForTest],
+});
+```
+
+This ensures that each test uses the same config for the `IonicModule` . 
+
+There might be files where `IonicModule` is used directly instead of `TestHelper.ionicModuleForTest` - this is a good chance to do some girl/boy scouting and fix it, but only if you are making changes to those files anyways.  
+
+#### The good test combines tests when appropiate
+
+In Kirby there is defined a [ `toHaveComputedStyle` ](https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/testing/element-css-custom-matchers.d.ts) custom matcher that checks the value of CSS properties an element is rendered with, like so: 
+
+``` 
+
+it('should render with correct border-width', () => {
+  expect(element).toHaveComputedStyle({
+    'border-width': '1px',
+  });
+});
+```
+
+When doing unit tests it can seem to be straightforward to create seperate tests for different CSS properties - we are only testing one thing per test after all. Consider the above example, where the only assessment is if `border-width` is correct. There might then be additional tests assessing if the element also has the correct `border-color` & `border-style` .  
+
+In that case, we would actually prefer if the test was created as a single _"should render with correct border"_ test; this is really what the reader of your test needs to know in the end. Here all three properties are assessed together like so:
+
+``` 
+
+it('should render with correct border', () => {
+  expect(element).toHaveComputedStyle({
+    'border-width': '1px',
+    'border-style': 'solid',
+    'border-color': 'transparent'
+  });
+});
+```
+
+This makes the intention more clear and also saves some time writing tests. Should the test fail due to one of the properties being rendered with an unexpected value; the error message will display which property is wrong. 
+
+We are aware that this requires a bit of gut feel. Testing for example if an element has `position: relative` & `background-color: #ffffff` would most likely not make sense. What would the test say? _Should have correct position and background-color_? There is really no relation between these two properties.
+
+Testing the properties `position` , `left` , `right` , `bottom` and `top` could make better sense as it could be tested as _"should be positioned correctly"_. As an additional example `font-family` , `font-size` , `font-weight` & `font-style` could be tested as _"should have correct typography"_.
+
+A rule of thumb is to test properties together whenever they have the same prefix such as `font-` . 
+This will however not always be the case as can be seen in the positioning example.
```

Comment:
Having a single rule of thumb example immediately followed by an exception makes the rule of thumb a bit weak 😊

Would it make sense to be more specific e.g. by mentioning that you probably often want to test individual properties of [shorthand properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#see_also) together?

For example `border` is shorthand for `border-width`, `border-style` and `border-color` so you would create a *border* test (just like the example above).

Similarly, `font` is also a shorthand property, which calls for at *font* test that combines assessment of individual font-* properties.

In addition to mentioning shorthand properties you could mention *related properties*, like the position/top/right/bottom/left example. Other examples of related properties are:

* **Color**: `color` and `background-color` which often *should* be tested together
* **Typography**: `font-*`, `line-height`, `letter-spacing` and `text-*` properties
* **Positioning**: `position`, `top/right/bottom/left` and `z-index`

Reply:
Let's have a chat about this one 😄